김나은/4월 3주차/BOJ_S1_2502_떡먹는호랑이.java

```diff
@@ -0,0 +1,36 @@
+package algo;
+
+import java.util.Scanner;
+
+public class BOJ_S1_2502_떡먹는호랑이 {
+
+	public static void main(String[] args) {
+
+		Scanner sc = new Scanner(System.in);
+		int D = sc.nextInt();
+		int K = sc.nextInt();
+
+		int[][] coefficient = new int[D+1][2];
+		coefficient[1][0] = 1;		coefficient[1][1] = 0;
+		coefficient[2][0] = 0;		coefficient[2][1] = 1;
```

Comment:
배열은 `0`으로 초기화 되어 있으니 값을 `0` 으로 초기화 하는 코드는 불필요한 것 같아요
```suggestion
		coefficient[1][0] = 1;
		coefficient[2][1] = 1;
```